checkov/openapi/checks/resource/v2/PathSchemeDefineHTTP.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+
+from typing import Any, Union, List
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.common.checks.enums import BlockType
+from checkov.openapi.checks.resource.v2.BaseOpenapiCheckV2 import BaseOpenapiCheckV2
+
+
+class PathSchemeDefineHTTP(BaseOpenapiCheckV2):
+    def __init__(self) -> None:
+        # https://learning.postman.com/docs/api-governance/api-definition/openapi2/#schemes-of-the-operation-have-http-scheme-defined
+        id = "CKV_OPENAPI_7"
+        name = "Ensure that the path scheme does not support unencrypted HTTP connection where all transmissions " \
+               "are open to interception- version 2.0 files"
+        categories = [CheckCategories.API_SECURITY]
+        supported_resources = ['security']
+        super().__init__(name=name, id=id, categories=categories, supported_entities=supported_resources,
+                         block_type=BlockType.DOCUMENT)
+
+    def scan_openapi_conf(self, conf: dict[str, Any], entity_type: str) -> tuple[CheckResult, dict[str, Any]]:
+        paths = conf.get("paths", {})
+        # if "paths" not in conf or not isinstance(paths, dict):
+        #     return CheckResult.FAILED, conf
```

Comment:
```suggestion
        paths = conf.get("paths")
        if not paths or not isinstance(paths, dict):
            return CheckResult.UNKNOWN, conf
```
lets go with that. If we don't have any paths or the config is broken, then we should return `unknown`, because we really don't know what is going on 🙂

Reply:
this should also fix the broken test

Reply:
thanks! I've changed it😃